plugins/module_utils/hmc_resource.py

```diff
@@ -519,6 +519,24 @@ def installOSFromNIM(self, loc_code, nimIP, gateway, lparIP, vlanID, vlanPrio, s
             " " + viosName + " " + profName + " " + systemName
         self.hmcconn.execute(lpar_netboot)
 
+    def installOSFromDisk(self, vios_iso, image_dir, vios_IP, gateway_ip, subnet_mask, mac_addr, system_name, vios_name, profile, label):
```

Comment:
change the parameter as below for better readablility
gateway_ip   =  vios_gateway
subnet_mask  = vios_subnetmask
Mac_addr = network_macaddr
and what is this label field?